src/main/java/io/outfoxx/swiftpoet/CodeWriter.kt

```diff
@@ -329,7 +329,9 @@ internal class CodeWriter constructor(
     for (i in typeSpecStack.indices.reversed()) {
       val typeSpec = typeSpecStack[i]
       if (typeSpec is ExternalTypeSpec) {
```

Comment:
Flatten into `if ((typeSpec is ExternalTypeSpec) && (typeSpec.name == simpleName)) {` ?

Reply:
I think it would work but I think maintaining the same style throughout the rest of the file for now.